### ISSUE TITLE
Prettier Logo

### ISSUE THREAD

**Issue by vramana** [prettier/prettier#757]
😄

**Comment by leipert** [comment]
Ideas could come from:
http://logojoy.com ( not affiliated, i like that one, you may need to remove the modal in the last step with an adblocker :D). Examples from there:

<img width="244" alt="screen shot 2017-02-21 at 23 18 10" src="https://cloud.githubusercontent.com/assets/2906107/23187795/7c136d0e-f88c-11e6-9695-e4648afb0376.png">
<img width="213" alt="screen shot 2017-02-21 at 23 18 35" src="https://cloud.githubusercontent.com/assets/2906107/23187806/86dcf82c-f88c-11e6-9070-39263d586596.png">
<img width="217" alt="screen shot 2017-02-21 at 23 18 46" src="https://cloud.githubusercontent.com/assets/2906107/23187808/89799540-f88c-11e6-9711-51f3b1f79c8f.png">

Would be happy to chime in if some funds are needed for a nice logo 
;0

**Comment by vjeux** [comment]
I really like this image I found on Google :p

![16700318_10154341665041188_7587041576571124754_o](https://cloud.githubusercontent.com/assets/197597/23230246/18186ae0-f8f8-11e6-9ddf-182dedaf32fa.png)

**Comment by rhengles** [comment]
How about something like this?

![Logo](http://i.imgur.com/JVBjaKR.png)

**Comment by ansumanshah** [comment]
Following the lines of @vjeux 
<img width="100" heigth="100" src="https://raw.githubusercontent.com/ansumanshah/prettier-brand/master/Prettier.png">
<img width="150" heigth="50" src="https://raw.githubusercontent.com/ansumanshah/prettier-brand/master/Badge.png">
https://github.com/ansumanshah/prettier-brand

**Comment by sheerun** [comment]
![](http://www.he-manreviewed.com/wp-content/uploads/2012/10/32-16-Knowing-Grin.png)

**Comment by ianstormtaylor** [comment]
Took a stab at something along the lines of what @vjeux was feeling...

![image](https://cloud.githubusercontent.com/assets/311752/24730511/e6690c38-1a18-11e7-95a9-2af91bb60440.png)
![image](https://cloud.githubusercontent.com/assets/311752/24730515/e86e0e70-1a18-11e7-9879-2f120949896a.png)

I put it all in [an organized repo](https://github.com/ianstormtaylor/prettier-logo), which I can transfer to the `prettier` org if you like it!

**Comment by kentcdodds** [comment]
That looks so awesome!

**Comment by vjeux** [comment]
@ianstormtaylor O M G, I love this logo <3

**Comment by vjeux** [comment]
@jlongster what do you think?

**Comment by ianstormtaylor** [comment]
Made a few tweaks to clean it up...

- Slightly increased the font size.
- Broke up the comment (gray) lines too.
- Aligned the P's stem.

![image](https://cloud.githubusercontent.com/assets/311752/24736977/385d81c8-1a40-11e7-8adc-c4c879ba6c31.png)
![image](https://cloud.githubusercontent.com/assets/311752/24736981/3a36c0e0-1a40-11e7-9418-9911e55211b6.png)

**Comment by jlongster** [comment]
I like @ianstormtaylor's logo a lot. I was going to propose my own but haven't had time and let's be honest, it's not going to be as high quality. 😛 

I think I like the way the gray lines are connected as single lines in the first version though. What do you think? It makes the P stand out better. And comments don't really differentiate their contents anyway; it's all just text.

Thanks a lot for all the logo ideas everyone!

**Comment by lydell** [comment]
I really like @ianstormtaylor’s logos. This is probably going to vary from person to person, and different monitors and stuff, but I just wanted to say that for me the dark variant way easier to “see” than the light variant.

In the dark one, it’s immediately obvious that there’s a “P” in there. In the light one, I’m like “there’s a bunch of colored lines there, it kinda looks like syntax highlighted code—oh wait there’s a P in there!”

Looking at the light one makes me think a bit of those [Ishihara test circles](https://en.wikipedia.org/wiki/Ishihara_test) (my eyes don’t fully pass such a test).

I quickly tried making the grey lines a bit lighter (`#E8EEF2`) and it seems to help. Not a designer, though ;)

Even with the grey lines completely removed, the dark one is still a little bit easier to see. The colors of the P score much higher contrast ratings against the dark background than against the light background according to this tool: http://leaverou.github.io/contrast-ratio/

End of potentially useless information :)

**Comment by ianstormtaylor** [comment]
Hey @jlongster sorry for the delay, I forgot about this issue.

Regarding the gray lines being connected, my personal opinion is that I prefer them disconnected, because I think it makes the logo feel more like the colored "P" happens to be built up of the components that were there. As opposed to feeling like it was placed there, and then the gray lines were added next to it—since without breaks they kinda feel like "speed lines". Here's what they look like side by side:

![image](https://cloud.githubusercontent.com/assets/311752/25016730/02c8c15c-2036-11e7-81ed-67b32f02d180.png)

But, I don't feel strongly, so I'm happy to use whichever version you want! Just let me know and I can make the switch back.

@lydell not useless, thank you! I agree with you, it's harder to see the "P" a bit on the white background. I've edited the colors slightly to make the purple and yellow slightly higher contrast, and I've made the gray lines in the white version lighter too. I think it helps a bit. It's still not as high contrast as the dark version, but I'm not sure we're going to get all the way there while maintaining the nice dark look, so at least it's slightly better. Thanks for the input!

@jlongster @vjeux I can transfer the `ianstormtaylor/prettier-logo` repo to the `prettier` organization if you'd like! I think I just need to be given "repo creation rights" just to be allowed to transfer, and then they can be removed. Let me know!
